docopt_value.h

```diff
@@ -105,7 +107,7 @@ namespace docopt {
 	};
 
 	/// Write out the contents to the ostream
-	std::ostream& operator<<(std::ostream&, value const&);
+	DOCOPT_API std::ostream& operator<<(std::ostream&, const value&);
```

Comment:
Please change back to `value const&` (the style for this library is east-const)

Reply:
Ok